utility_scripts/deploy/deploy_hyperion.py

```diff
@@ -128,13 +131,17 @@ def get_hyperion_release_dir_from_args(repo: repo) -> str:
 Only do so if you have informed the beamline scientist and you're sure Hyperion is not running.
 """
     )
-    # Creates symlink: software/bluesky/hyperion_version -> software/bluesky/hyperion
+    # Creates symlinks: software/bluesky/hyperion_latest -> software/bluesky/hyperion_{version}/hyperion
+    #                   software/bluesky/hyperion -> software/bluesky/hyperion
     if move_symlink == "y":
```

Comment:
Should: I wonder if we should have another step here that says something like "Previous live version was X, do you want to set this is the stable version?" Then move it to `hyperion_stable`?